app/views/application_mailer/provider_linked.html.haml

```diff
@@ -5,10 +5,10 @@
   We're contacting you to let you know that your account at #{link_to Translation.translate('City of Boston DND Warehouse'), root_url} has been signed into with #{@provider_name} for the first time.
 
 %p.content
-  You should now use #{@provider_name} to sign in, your original password has been disabled.
+  You should now use #{@provider_name} to sign in.
```

Comment:
This message is confusing because the user might not have created a temporary password